ui/message.lua

```diff
@@ -3,6 +3,16 @@ local S = minetest.get_translator("mail")
 
 local FORMNAME = "mail:message"
 
+local function interleaveMsg(body)
+	local lines = (body or ""):split("\n", true)
+	local il_lines = {}
+	for _,l in ipairs(lines) do
+		il_lines[#il_lines+1] = "> " .. l
+	end
+	return table.concat(il_lines, "\n")
```

Comment:
Fast accept :)
I think it'll leave just `> ` if body is empty string, but then does it matter? If new mail also uses this then it might be a minor annoyance and could use check like
```lua
return body and #body > 0 and "> " .. (body or ""):gsub("\n", "\n> ") or ""
```

Reply:
Mail actually can't be sent when body is empty, so is it necessary to check ?

Reply:
> Mail actually can't be sent when body is empty, so is it necessary to check ?

Checks are not really necessary but can affect some cases: if body is empty it will output `> ` but I think it is only used for replies / forwards which should have non empty body and in that case checks are not needed.

Anyway, won't break anything. Checks are only needed if you want output to always be an empty string if input is also empty string.

For example, without checks following expressions are true:
```lua
local function interleaveMsg(body)
	return "> " .. (body or ""):gsub("\n", "\n> ")
end

interleaveMsg(nil) == "> "
interleaveMsg("") == "> "
interleaveMsg(" ") == ">  "
interleaveMsg("foobar") == "> foobar"
interleaveMsg("foo\nbar") == "> foo\n> bar"
```

For example, with checks following expressions are true:
```lua
local function interleaveMsg(body)
	return body and #body > 0 and "> " .. body:gsub("\n", "\n> ") or ""
end

interleaveMsg(nil) == ""
interleaveMsg("") == ""
interleaveMsg(" ") == ">  "
interleaveMsg("foobar") == "> foobar"
interleaveMsg("foo\nbar") == "> foo\n> bar"
```